webui/react/src/components/kit/Icon.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { CSSProperties } from 'react';
+
+import Tooltip from 'components/kit/Tooltip';
+
+import css from './Icon.module.scss';
+
+export const IconSizeArray = [
```

Comment:
Which of these are we using now?

Reply:
Ah, I see. From a quick regex search I found:

- Tiny
- Small
- Medium
- Large
- Big
- Mega

Which means that we're not using:

- Great
- Huge
- Enormous
- Giant
- Jumbo

Some of the components using Icons allowed for passing in arbitrary sizes, and I tried to track down everything that we pass in, but I might have missed a couple. My guess is that we needed the Mega size icon at one point and just expanded our sizing scale up to it without using the sizes in between.